### DISCUSSION TITLE
Is there a reason a link to myabandonware is not included?

### DISCUSSION THREAD

**Discussion by UnconnectedBedna** [openblack/openblack#813] [Q&A] [answered]
I bumped into this project on accident by watching a youtube, and I saw there was mentions of arch linux when he scrolled by the gitbub page...

On the aur page there is a comment with a link to myabandonware saying it is now released and it is fully legal do download through the link.
I am not providing the link here if it actually still is illegal to download there...

Otherwise, is there a reason this is not mentioned on the readme.md?

**Comment by handsomematt** [comment, marked as answer]
I'm not sure why people think abandonware = fully legal

We want to keep ourselves fully distant from redistributing game assets, same as any other project of this nature

**Comment by UnconnectedBedna** [comment]
Well, you would not be distributing anything, I was asking about providing a link, but I got my answer.
